methods/incidentReports.coffee

```diff
@@ -35,18 +47,20 @@ Meteor.methods
       Meteor.call("editUserEventLastIncidentDate", incident.userEventId)
     return incident._id
 
-  editIncidentReport: (incident) ->
+  editIncidentReport: (incident, userEventId) ->
```

Comment:
I think either file is fine.

Querying for the associated eventIds would be better that passing in the id. Since the incident may be shared by multiple events, events other than just the passed in event could need to have their meta-data updated.